flocker/node/testtools.py

```diff
@@ -248,7 +248,7 @@ def ideployer_tests_factory(fixture):
 
     :return: ``TestCase`` subclass that will test the given fixture.
     """
-    class IDeployerTests(TestCase):
+    class IDeployerTests(AsyncTestCase):
```

Comment:
You may need to update the fixture implementation to use `test.reactor` instead of global reactor.